models.js

```diff
@@ -9,19 +9,36 @@ exports.selCategories = (req,res) => {
     });
 }
 
-exports.selReviews = () => {
-    return db.query(`
-    SELECT reviews.review_id, reviews.owner, reviews.title, reviews.category, reviews.review_img_url, reviews.created_at, reviews.designer, reviews.votes,
-    COUNT(comment_id) AS comment_count
-    FROM reviews
-    LEFT JOIN comments
-    ON comments.review_id = reviews.review_id
-    GROUP BY reviews.review_id
-    ORDER BY created_at DESC;`)
-    .then(result => {
-        return result.rows;
-    })
+exports.selReviews = (category = undefined, sortOn = 'created_at', order = 'DESC') => {
+    if (category !== undefined) {
```

Comment:
you could do this by building up the query over time, it totally works though so I wouldn't worry about it